features/media-queries-prefers-reduced-data.yml

```
@@ -0,0 +1,6 @@
+name: prefers-reduced-data
+description: The `prefers-reduced-data` CSS media feature detects whether the user has a preference for using less internet traffic.
```

```suggestion
description: The `prefers-reduced-data` CSS media feature sets styles based on whether the user prefers to consume less internet traffic.
```

It would be good to  give a short example of when a web developer might want to use this. MDN has a nice example about using a system font rather than loading a custom web font. Maybe add "For example, you can use this media feature to avoid loading large font files and use a system font instead."